Submitted By: Mario Fetka (mario dot fetka at gmail dot com)
Date: 2008-08-30
Initial Package Version: 1.1.5
Origin: me
Upstream Status: Gentoo specific
Description: correct filelocations
 
diff -Naur fedora-ds-admin-1.1.5.orig/admserv/cfgstuff/httpd-2.2.conf.in fedora-ds-admin-1.1.5/admserv/cfgstuff/httpd-2.2.conf.in
--- fedora-ds-admin-1.1.5.orig/admserv/cfgstuff/httpd-2.2.conf.in	2007-08-01 18:14:53.000000000 +0000
+++ fedora-ds-admin-1.1.5/admserv/cfgstuff/httpd-2.2.conf.in	2008-08-30 12:25:00.766003282 +0000
@@ -141,10 +141,10 @@
 LoadModule disk_cache_module @moddir@/mod_disk_cache.so
 LoadModule file_cache_module @moddir@/mod_file_cache.so
 LoadModule mem_cache_module @moddir@/mod_mem_cache.so
-LoadModule cgi_module @moddir@/mod_cgi.so
-LoadModule restartd_module     @admmoddir@/mod_restartd.so
-LoadModule nss_module         @nssmoddir@/libmodnss.so
-LoadModule admserv_module     @admmoddir@/mod_admserv.so
+LoadModule cgid_module @moddir@/mod_cgid.so
+LoadModule restartd_module     @moddir@/mod_restartd.so
+LoadModule nss_module         @moddir@/mod_nss.so
+LoadModule admserv_module     @moddir@/mod_admserv.so
 
 ### Section 2: 'Main' server configuration
 #
diff -Naur fedora-ds-admin-1.1.5.orig/admserv/cfgstuff/httpd.conf.in fedora-ds-admin-1.1.5/admserv/cfgstuff/httpd.conf.in
--- fedora-ds-admin-1.1.5.orig/admserv/cfgstuff/httpd.conf.in	2007-08-01 18:14:53.000000000 +0000
+++ fedora-ds-admin-1.1.5/admserv/cfgstuff/httpd.conf.in	2008-08-30 12:26:03.402003388 +0000
@@ -140,10 +140,10 @@
 LoadModule disk_cache_module @moddir@/mod_disk_cache.so
 LoadModule file_cache_module @moddir@/mod_file_cache.so
 LoadModule mem_cache_module @moddir@/mod_mem_cache.so
-LoadModule cgi_module @moddir@/mod_cgi.so
-LoadModule restartd_module     @admmoddir@/mod_restartd.so
-LoadModule nss_module         @nssmoddir@/libmodnss.so
-LoadModule admserv_module     @admmoddir@/mod_admserv.so
+LoadModule cgid_module @moddir@/mod_cgid.so
+LoadModule restartd_module     @moddir@/mod_restartd.so
+LoadModule nss_module         @moddir@/mod_nss.so
+LoadModule admserv_module     @moddir@/mod_admserv.so
 
 ### Section 2: 'Main' server configuration
 #
